transports/rabbitmq/callbacks_queue_rabbit_[,3].partial.md

```diff
@@ -1,6 +1,8 @@
-When scaling out an endpoint, any of the endpoint instances can consume messages from the same shared broker queue. However, this behavior can cause problems when dealing with callback messages because the reply message for the callback needs to go to the specific instance that requested the callback.
+When sending a message, a callback can be registered that will be invoked when a response arrives.
```

Comment:
Is this different from the callbacks package? Why do we have both? Should we differentiate them in doco?

Reply:
Nope. I copied the header from that callback page to have a nice intro here. I think the old one suggested that `callback` *is* the message while the main callbacks page https://docs.particular.net/nservicebus/messaging/callbacks makes it clear that the callback is registered for handling a response.